test/EndToEnd/Apps/CSharpApp/StateInputBinding/StateInputBinding.cs

```diff
@@ -0,0 +1,26 @@
+namespace bindings_state
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Dapr;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Extensions.Logging;
+
+    public static class StateInputBinding
```

Comment:
Shall we exclude these from code coverage?

Reply:
Good idea, added some ignore paths to our YAML.